be/src/storage/storage_engine.cpp

```diff
@@ -714,9 +714,37 @@ Status StorageEngine::_start_trash_sweep(double* usage) {
     // clean unused rowset metas in KVStore
     _clean_unused_rowset_metas();
 
+    _do_manual_compact();
+
     return res;
 }
 
+void StorageEngine::_do_manual_compact() {
```

Comment:
How long will this manual compact() last?
Will it stall the load and other operations?

Reply:
About a few dozen seconds approximately.
No, it will not.